src/ArticleFilter.m

```diff
@@ -69,7 +69,10 @@ +(BOOL)lastRefreshFilterComparator:(Article *)theArticle
  */
 +(BOOL)twoDaysFilterComparator:(Article *)theArticle
 {
-	return ([theArticle.date compare:[[NSCalendarDate date] dateByAddingTimeInterval:-172800]] != NSOrderedAscending);
+    return [theArticle.date compare:[NSCalendar.currentCalendar dateByAddingUnit:NSCalendarUnitDay
+                                                                           value:-2
+                                                                          toDate:theArticle.date
+                                                                         options:0]] != NSOrderedAscending;
 }
```

Comment:
I made a mistake there, should have passed a new date object in `toDate` rather than the `theArtice.date`. I kept the NSCalendar implementation, because it is semantically a little clearer, I think.